resources/authgear/templates/en/web/authflowv2/__password_strength_meter.html

```diff
@@ -0,0 +1,13 @@
+{{ define "authflowv2/__password_strength_meter.html" }}
+<div class="{{ $.Classname }} password-strength-meter" data-password-policy-target="currentMeter">
```

Comment:
In the original implementation, we use `<meter>` element. Any specific reason we did not follow?

Make use of `<meter value>` in the implementation. The job of the controller is to assign correct `value`. The style should be written against `<meter value>`. I mean, you can write selector that looks like `password-strength-meter[value=0] .password-strength-meter__someelement`

Reply:
But the new design is actually not a meter, ui wise 🤔

Reply:
Visually it does not look like a meter, but semantically it is. We can use `appearance: none` to make it look like a div, and apply our design on it.